frontend/src/components/landing/Landing.tsx

```diff
@@ -207,6 +208,44 @@ const Landing = () => {
         </div>
       </div>
 
+      <div className={styles.channelContainer}>
+        <div className={styles.innerContainer}>
+          <div className={styles.channelImg}>
+            <img src={channelsImg} alt="Channels" />
+          </div>
+          <div className={styles.channelDescription}>
+            <span>CHANNELS</span>
+
+            <h4 className={styles.new}>Telegram</h4>
+            <p>
+              Telegram is the #1 messaging app for Singaporeans aged 35 and
+              under. Engage your subscribers regularly through our Telegram bot
+              broadcasting feature. You can easily send private messages to your
+              agency employees for a fitness event or engage citizens for a
+              donation drive.
+            </p>
+
+            <h4>Email</h4>
+            <p>
+              Go digital and send personalized emails. No more letterheads and
+              stamps. Use our password-protected feature to send sensitive
+              information like a payslip or personal PIN to your recipients.
+            </p>
+
+            <h4>SMS</h4>
+            <p>
+              Move away from calling. Send transactional SMS easily through our
+              Twilio integration. Remind citizens to come for their
+              appointments. The possibilities are endless.
+            </p>
+
+            <PrimaryButton className={styles.button}>
+              Learn More <i className="bx bx-right-arrow-alt"></i>
```

Comment:
@NatMaeTan @sarahsalim what is this the correct hover state for the "Learn more" button for the channels section of the landing page? Looks a little hard to read.

![image](https://user-images.githubusercontent.com/3666479/100411107-b63e2900-30ab-11eb-992f-5a42d306142a.png)